Server/mods/deathmatch/logic/CStaticFunctionDefinitions.cpp

```diff
@@ -333,14 +333,30 @@ bool CStaticFunctionDefinitions::DestroyElement(CElement* pElement)
     {
         return false;
     }
+    // Its team trigger onPlayerTeamChange for each player in the team
+    else if (iType == CElement::TEAM) 
+    {
```

Comment:
Here you can remove the "else" part because it's kind of implicit. And put a blank line at 336 to seperate both conditions
```suggestion
    {
        return false;
    }
    
    // Its team trigger onPlayerTeamChange for each player in the team
    if (iType == CElement::TEAM) 
    {
```

Reply:
yeah, i will make it better